table/partitioned_filter_block_test.cc

```diff
@@ -66,12 +66,13 @@ class PartitionedFilterBlockTest : public testing::Test {
   }
 
   uint64_t MaxFilterSize() {
+    uint32_t dont_care1, dont_care2;
```

Comment:
The function CalculateSpace returns three values, one the normal returned value which is the calculated space and two others returned using by reference values passed to the function. Here we do not care about the latter two so we ignore their updated value.

About Line 71, we are sure about the type but it is better to have a safety check for future code changes. I will add an assert for null checking.